docs/source/user_guides/basic_conceptions.rst

```diff
@@ -139,165 +148,185 @@ That's what the agent consists of:
   no transition for the message given by user is found.
```

Comment:
I think it would be more clear this way:
```
* ``fallback_node`` is an "exception handling node"; user will be transferred here if
  no ``TRANSITIONS`` condition is satisfied.
```

Reply:
Updated